blazor/datagrid/dialog-editing.md

```diff
@@ -136,4 +136,101 @@ In the below example we have changed the dialog's header text and footer button
 }
 ```
 
-> You can refer to our [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore our [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
```

Comment:
This line seems to be repeated below, is it by design?

Reply:
@@ -136,4 +136,101 @@ In the example below, we have changed the dialog's header text and footer button: